src/test/java/org/mockitousage/bugs/GenericsMockitoAnnotationsTest.java

```diff
@@ -0,0 +1,36 @@
+package org.mockitousage.bugs;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.Mock;
+
+import java.util.ArrayList;
+import java.util.Collection;
+
+import static org.mockito.BDDMockito.given;
+import static org.mockito.MockitoAnnotations.initMocks;
+
+/**
+ * This was an issue reported in #1174.
+ */
+public class GenericsMockitoAnnotationsTest {
+
+    @Mock
+    private TestCollectionSourceProvider testCollectionSourceProvider;
+
+    @Test
+    public void test() {
```

Comment:
I would name it in more meaningful way (e.g. should not fail with ... )

Anyway, it seem to be work on Travis.

Reply:
Ah yeah good catch. Will update!